spec/system/candidate_interface/entering_details/candidate_entering_other_qualification_btec_spec.rb

```diff
@@ -0,0 +1,96 @@
+require 'rails_helper'
+
+RSpec.feature 'Entering their other qualifications' do
+  include CandidateHelper
+
+  scenario 'Candidate submits their BTEC qualification' do
+    given_i_am_signed_in
+    and_i_visit_the_site
+
+    when_i_click_on_other_qualifications
+    then_i_see_the_select_qualification_type_page
+
+    when_i_select_add_an_other_uk_qualification
+    and_i_fill_in_the_qualification_name_as_btec
+    and_i_click_continue
```

Comment:
Suggestion - merge the above 3 into something like `when_i_attempt_to_add_a_btec`